backfiller/backfiller/main.py

```diff
@@ -269,7 +269,7 @@ def get_nodes(self):
 				WHERE backfill_from""")
 			for row in results:
 				nodes[row.name] = row.url
-
+		nodes = {key:val for key, val in nodes.items() if val != self.localhost}
```

Comment:
so now we're both removing things where the url == self.localhost AND where the name == self.localhost. so is self.localhost meant to be a name or a url? I'm confused.

Reply:
I think we should stick to node "names" as identifiers rather than URLs, especially since many different URLs can actually mean the same thing, eg. `http://example.com` vs `http://example.com/` vs `http://example.com:80/`

Reply:
Ok fair enough. I'll change the documentation to reflect this.